README.md

```diff
@@ -4,19 +4,50 @@
 
 This connector is an early experiment to read from/ write to a mongo database 
```

Comment:
I realize this pre-dates the changes in this PR, but we're at a state now where we can also update this sentence a bit

```suggestion
Read and write data to MongoDB with Dask
```